src/Umbraco.Web.Common/Runtime/AspNetCoreBootFailedComposer.cs

```diff
@@ -7,7 +7,6 @@ namespace Umbraco.Web.Common.Runtime
     /// <summary>
     /// Executes if the boot fails to ensure critical services are registered
     /// </summary>
-    [RuntimeLevel(MinLevel = RuntimeLevel.BootFailed)]
     public class AspNetCoreBootFailedComposer : IComposer
```

Comment:
Same for this composer: it should only be added when the application boot failed.

Reply:
Min level starts at boot failed and gets bigger, if it were supposed to stop it would use max level.

This is no longer handled in service registration, runtime state should be checked at resolve time not at ref time (which the boot failed middleware already does)